packages/app/src/systems/Core/components/Layout/TopBar.tsx

```diff
@@ -150,18 +122,48 @@ export function TopBar({ type = TopBarType.internal, ...props }: TopBarProps) {
 const styles = {
   root: cssObj({
     py: '$2',
-    px: '$4',
+    px: '$5',
     gap: '$3',
     alignItems: 'center',
     minHeight: '50px',
-    boxShadow: '$sm',
     transition: 'none',
-    background:
-      'linear-gradient(268.61deg, rgba(255, 255, 255, 0) 0%, rgba(255, 255, 255, 0.02) 87.23%)',
+  }),
+  topbarIcon: cssObj({
+    px: '$0 !important',
+    color: '$intentsBase8 !important',
+    transition: 'color 0.2s ease-in-out',
+
+    '&:hover': {
+      color: '$intentsBase11 !important',
+    },
+  }),
+  title: cssObj({
+    textSize: 'lg',
+    fontWeight: '$normal',
+    color: '$intentsBase12',
+  }),
+  container: cssObj({
+    position: 'relative',
+    pl: '$6',
+    gap: '$3',
+    alignItems: 'center',
+    flex: 1,
 
     '&[data-home="true"]': {
-      boxShadow: '$none',
-      background: 'transparent',
+      pl: '$0',
     },
   }),
+  backIcon: cssObj({
+    position: 'absolute',
+    top: '50%',
+    left: '$0',
+    px: '$2 !important',
+    width: '$4 !important',
+    transform: 'translateY(-50%)',
+
+    '&:not([aria-disabled=true]):active, &:not([aria-disabled=true])[aria-pressed=true]':
+      {
+        transform: 'scale(0.97) translateY(-50%)',
+      },
```

Comment:
Should we have this inside the fuel-ui? Looks like a issue

Reply:
It's because I was overlapping the `transform` in the element, so I need to reconfigure it, but it doesn't need to be inside the `fuel-ui`